### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -L crate,ser,fo,noo,nd,cappable,pard,te,ans,succcess`

### CODE REVIEW

**Review by Turakar — APPROVED** [review]
I reviewed all the changes from a language perspective (i.e. syntactically and grammatically) and found no mistakes.

*Disclaimer: I am not a Rocket team member.*

File: CHANGELOG.md [hunk 1]
```diff
@@ -2054,7 +2054,7 @@ In addition to new features, Rocket saw the following smaller improvements:
 
 ## Core
 
-  * Typo in `Outcome` formatting fixed (Succcess -> Success).
```
Comment (jinakola) [inline CHANGELOG.md:2057]: Found this same line "fixed" in #2353 and really don't know if it should be changed

Comment (kianmeng) [reply CHANGELOG.md]: @jinakola Fixed.

**Comment by SergioBenitez** [comment]
This is awesome! Thank you. Merging this soon.

**Comment by SergioBenitez** [comment]
Merged, thank you!

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request closed without merging**